actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -245,7 +247,7 @@ def tag_string(name, content = nil, options, escape: true, &block)
           content_tag_string(name, content, options, escape)
         end
 
-        def self_closing_tag_string(name, options, escape = true, tag_suffix = " />")
+        def self_closing_tag_string(name, options, escape = true, tag_suffix = (void_element_trailing_slash == false ? ">" : " />"))
```

Comment:
I'm not sure I understand why this change is being made. Is this method being used somewhere outside of `define_self_closing_element` and `define_void_element`? The idea in 343897980cf9c1a8c1d102df3afb0631f5c9145b was that this method is for self closing elements but can be specialized for void elements by passing in the suffix.

Since omitting the `/` is _not_ valid for self-closing elements, I'm not sure it should ever be the default for this method.

Reply:
This method is used for both: void elements and the self closing SVG tags.

I moved the configuration check to the void element definition method and changed the default value to the tag closing without the ending slash at dcf7b87. This is because going forward, the ending slash should be rather an exception (i.e. SVGs) than the default.

Reply:
> This is because going forward, the ending slash should be rather an exception (i.e. SVGs) than the default.

Right, I don't think this is a good idea. Void elements are a special case of self-closing tags where the `/` is optional instead of required. If the default is `>` then every caller wanting a self-closing tag _must_ pass in `/>` to get a valid tag. If the default is `/>` then passing in `>` is just an optimization for void tags but the method won't generate invalid HTML.

Reply:
The default can be changed back as per this request but I'll wait for the answer to the other question before doing that.